lib/compat/wordpress-5.9/move-theme-editor-menu-item.php

```diff
@@ -9,7 +9,7 @@
  * Moves the "theme editor" under "tools" in FSE themes.
```

Comment:
Should this be "block" themes?
```suggestion
 * Moves the "theme editor" under "tools" in block themes.
```